docs/modules/multi-tenancy/proc-multi-tenancy.adoc

```diff
@@ -0,0 +1,43 @@
+// file included in the following:
+//
+// assembly-multi-tenancy-filter.adoc
+
+[id='proc-multi-tenancy-{context}']
+= (Preview) Setting up the Multi-tenancy filter
+
+[role="_abstract"]
+This procedure describes how to set up the Multi-tenancy filter by configuring it in Kroxylicious.
+The filter dynamically prefixes resource names to create isolation between tenants using the same Kafka cluster.
+The prefix representing a tenant is taken from the name of the virtual cluster representing the tenant. 
+For example, if the virtual cluster is named `tenant-1`, the prefix is `tenant-1`.
+Each tenant must be represented by a unique virtual cluster that points to the same cluster.
```

Comment:
Virtual cluster names have to be globally unique so we can't have `tenant-1 -> kafkaA` & `tenant-1 -> kafkaB` within kroxylicious config model but this sentence implies that that would be supported.

Reply:
Changed as follows:

```
Each tenant must be represented by a unique virtual cluster, and virtual cluster names must be globally unique within the Kroxylicious configuration. 
This means that the same virtual cluster name cannot be used to represent different Kafka clusters.
```